core/modules/filter/filter.module

```diff
@@ -1159,6 +1159,11 @@ function filter_process_format($element) {
     $element['#format'] = filter_default_format($user);
   }
 
+  // Remove the Plain text format if not set and other options are available.
+  if (($element['#format'] != 'plain_text') && (count($options) > 1) && array_key_exists('plain_text', $options)) {
```

Comment:
Rather than checking for the string "plain_text", this should instead check the variable (the fallback format is configurable):

```
$fallback_format = config_get('system.core', 'filter_fallback_format');
```